Common/Product/SharedProject/CommonProjectNodeProperties.cs

```diff
@@ -427,7 +429,7 @@ internal CommonProjectNodeProperties(ProjectNode node)
 
         [Browsable(false)]
         public string __id {
-            get { throw new NotImplementedException(); }
+            get => __idStr;
```

Comment:
This fixes a crash I was having when trying to use the Quick find thing at the top of VS.